src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/AStar.java

```diff
@@ -0,0 +1,100 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Map;
+
+public class AStar {
+    private static final NodeBase nodeBase = NodeBase.getInstance();
+
+  public static double getDistanceFromCurrentNodeToEndOrStartNode(NodeArm point, NodeArm current){
+       return Math.abs(nodeBase.getDistanceBetweenToPoints(point,current));
```

Comment:
Distance is usually defined as being absolute, is the inner method (`getDistanceBetweenToPoints`) ever called without using `Math.abs` around it?

Reply:
now no butt if we want the arm to move by points and not speed we need it not absolute